src/tests/ftest/pool/storage_ratio.yaml

```diff
@@ -0,0 +1,50 @@
+server_manager:
+    srv_timeout: 500
+hosts:
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+timeout: 900
+server_config:
+  servers_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:81:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["0000:da:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+pool:
+    mode: 146
+    name: daos_server
+    svcn: 1
+    control_method: dmg
+    stoage_ratio:
+#   - [scmsize, nvmesize, Expected Result]
+    - ['70G', '100G', 'PASS']  # SCM Size ratio is greater than 6%
+    - ['91G', '650G', 'PASS'] # SCM Size ratio is greater than 6%
+    - ['10G', '1G', 'FAIL']    # Low NVMe Size
+    - ['16', '100G', 'FAIL']   # Low SCM Size
```

Comment:
Do we expect pool creation fail for non-optimal ratio? I think dmg only print a warning message for such case.

Reply:
I think so. There was 6% ration we were maintaining. DAOS-4814 open to put it back.

Reply:
creation will not fail for a non-optimal ratio, it will only provide a warning. the feature of failing pool create due to a non-optimal storage size ratio has not been requested and I don't think it in any requirements document. @johannlombardi please confirm

Reply:
Sorry not able to follow the comment. 
Line number 46/47 is creating the very low size of SCM (16 Bytes) and NVMe (1G), which is going to fail.
Now if user will create the pool with less than 1% it will not fail but just the warning ?

Reply:
the only thing that is triggered by the low ratio (<1%) is a warning, no failure will occur with a low ratio

Reply:
So in that case we need to change the tests to verify that warning is getting generated